[core]/es_extended/config.lua

```diff
@@ -60,7 +60,7 @@ Config.RemoveHudCommonents = {
 	[22] = false, --HUD_WEAPONS
 }
 
-Config.MaxAdminVehicles = false -- admin vehicles spawn with max vehcle settings
+Config.MaxAdminVehicles = true -- admin vehicles spawn with max vehcle settings
```

Comment:
SpawnVehMaxUpgrades i think it's better variable name.

Reply:
Updated